articles/how-to-guides/create-an-application-template/create-an-application-template.md

```diff
@@ -2,3 +2,83 @@
 uid: how-to-guides.create-an-application-template
```

Comment:
Similar to [my comment](https://github.com/IntentSoftware/Docs/pull/5#issuecomment-853922107) on the review, we could add a section to the Application Templates article in references to this how-to.